iothub/service/src/Feedback/MessageFeedbackProcessorClient.cs

```diff
@@ -98,7 +98,7 @@ protected MessageFeedbackProcessorClient()
         /// <exception cref="IotHubServiceException"> with <see cref="HttpStatusCode.RequestTimeout"/>If the client operation times out before the response is returned.</exception>
         /// <exception cref="OperationCanceledException">If the provided cancellation token has requested cancellation.</exception>
         /// <exception cref="SocketException">If a socket error occurs.</exception>
-        /// <exception cref="WebSocketException">If an error occurs when performing an operation on a WebSocket connection.</exception>
+        /// <exception cref="WebSocketException">If an error occurs when performing an operation on a web socket connection.</exception>
         /// <exception cref="IOException">If an I/O error occurs.</exception>
```

Comment:
I feel like all these (not ArgumentException, ArgumentNullException, InvalidOperationException, ObjectDisposedException) should be wrapped in an IotHubServiceException. @brycewang-microsoft

Reply:
This appears overwhelmed by lots of PR comments so I just notice it. I am looking into this.